ovirt-engine-dwh.spec.in

```diff
@@ -20,14 +20,20 @@
 %global systemd systemd
 %global commons_collections apache-commons-collections
 
+%define rhv_build 0
 %if 0%{?rhel}
     %if 0%{?centos}
-        %global dom4j ovirt-engine-wildfly
+        %define rhv_build 0
     %else
-        %global dom4j eap7-dom4j
+        %define rhv_build 1
     %endif
 %endif
 
+%global dom4j ovirt-engine-wildfly
+%if 0%{rhv_build}
```

Comment:
Not sure if we need the '0' here

Reply:
`0` is not needed as default value is provided on line 23